### PR TITLE
Fix for fabriclocation model update plus other miner renaming

### PR DESCRIPTION
**Pull request by LingyunSu**
Fix for fabriclocation model update plus other miner renaming

### CODE REVIEW

**Comment by LingyunSu** [comment]
@bganapa @JesusAlvarezTorres @vikasnav

**Comment by bganapa** [comment]
@LingyunSu Please change the Assembly version in assemblyinfo.cs, nuget version in nuget package and add release notes

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @LingyunSu , could you
- Add a link to the swagger spec review PR in the description
- Include the `.txt` file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Update versions, tags, if necessary

Thanks

**Comment by LingyunSu** [comment]
Hey @isra-fel:
- This is the swagger: [FabricLocation.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json)

- There is no change on the RP file:
  https://github.com/Azure/azure-sdk-for-net/blob/master/eng/mgmt/mgmtmetadata/azsadmin_resource-manager_fabric.txt

- Version, release note has been updated.

@bganapa

File: sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -15,5 +15,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("2.9.0.0")]
-[assembly: AssemblyFileVersion("2.9.0.0")]
+[assembly: AssemblyVersion("2.9.1.0")]
```
Comment (isra-fel) [inline sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyVersion("2.9.0.0")]
```
AssemblyVersion only needs to be updated when there's a major version update.

File: sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/tests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -15,5 +15,5 @@
 
 [assembly: Guid("1b24bc4a-6e4d-4feb-b5d5-c9684c60a308")]
 
-[assembly: AssemblyVersion("2.9.0.0")]
-[assembly: AssemblyFileVersion("2.9.0.0")]
+[assembly: AssemblyVersion("2.9.1.0")]
```
Comment (isra-fel) [inline sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/tests/Properties/AssemblyInfo.cs]: Updating the version of test proj is not required (you can do it anyway)

**Review by isra-fel — CHANGES_REQUESTED** [review]
There's no way the metadata file was not changed -- you are generating from a new commit of the spec, so the "commit" section must have been changed.
Did you generate the source code by running `generate.ps1`?

File: sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (LingyunSu) [reply sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs]: We have a small bug fix against the previous one.  We need this update if people installed the old version. @bganapa is that right?

**Comment by LingyunSu** [comment]
@bganapa @isra-fel @JesusAlvarezTorres @vikasnav  txt file updated.  @isra-fel: remember there is an issue in generate script, the txt landing in a wrong place.

File: sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (isra-fel) [reply sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs]: Updating `AssemblyVersion` doesn't help the user to upgrade -- user who was using the old version can still build correctly.

Comment (isra-fel) [reply sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs]: Here's a great ariticle talking about it: https://support.microsoft.com/en-us/help/556041

**Comment by isra-fel** [comment]
> @bganapa @isra-fel @JesusAlvarezTorres @vikasnav txt file updated. @isra-fel: remember there is an issue in generate script, the txt landing in a wrong place.

I tried it locally and the metadata was generated in the correct folder 🤔 
One thing you could try is to rerun
`msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`
to install the generate cmdlet again

**Review by isra-fel — APPROVED** [review]

**Pull request merged**